lib/api/users/user.ts

```diff
@@ -0,0 +1,27 @@
+import { db} from "../../db";
+
+export function getlockers() {
+    return db.prepare("SELECT * FROM lockers").get()
+}
+
+export function register(building: number, number: number, name: string) {
+
+    const user_statement = db.prepare("INSERT INTO users VALUES(?)").run(name)
+    const statement = db.prepare("INSERT INTO registrations (building_id, num, reported_at, user_id) VALUES (?, ?, ?, ?)")
+
+    const info = statement.run(building, number, Date.now(), user_statement.lastInsertRowid)
+}
+
+
+export function deregister(building: number, number: number) {
```

Comment:
We need to make sure that the user must know the name and email of the locker they're deregistering